### PR TITLE
Add ActivityPub

### LINKED ISSUE #7483: Icon Request: Fediverse
### Brand Name

Fediverse

### Website

https://en.wikipedia.org/wiki/Fediverse

### Popularity Metric

There is no ranking as such, because it is a logo of a community of many different softwares (mastodon, pleroma, GNU Social, etc). The logo proposal: https://socialhub.activitypub.rocks/t/fediverse-logo-or-icon-proposal/1057

### Official Resources for Icon and Color

https://upload.wikimedia.org/wikipedia/commons/9/93/Fediverse_logo_proposal.svg 

### Additional Comments

I have seen that there are different icons for Mastodon, GNU Social, Pleroma, etc. Perhaps it would be interesting to add the logo of the "Fediverse" that encompasses all these softwares that use federated protocols.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
activitypub

**Issue:** closes #7483, compliment to #9685

**Similarweb rank:** [similarweb.com/website/activitypub.rocks](https://www.similarweb.com/website/activitypub.rocks) - #760834

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Adds ActivityPub (protocol) icon. This is the network that powers Mastodon, Firefish, Pleroma, Misskey, etc which already have icons in SimpleIcons and can be used as a protocol discriminator for a given network.

SVG size is 235 bytes.

### CODE REVIEW

**Comment by ThatOneCalculator** [comment]
As per advice given on Discord, the Fediverse icon will be a separate PR

File: icons/activitypub.svg
Comment (PratyushJayachandran) [inline icons/activitypub.svg:1]: I see [some differences!](https://github.com/simple-icons/simple-icons/assets/35000807/7c048a71-fbb6-4338-93df-4680f491a71d)

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
The JSON data is correct. but the icon has minor issues when comparing to the original. Can you take another look @ThatOneCalculator?

File: icons/activitypub.svg
Comment (ThatOneCalculator) [reply icons/activitypub.svg:1]: Probably SVGOMG again ^^;

**Review by PratyushJayachandran — APPROVED** [review]
Perfect match @ThatOneCalculator 🎉

**Pull request merged**